plugin-gradle/src/main/java/com/diffplug/gradle/spotless/KotlinExtension.java

```diff
@@ -25,18 +25,20 @@
 import com.diffplug.spotless.kotlin.KtLintStep;
 
 public class KotlinExtension extends FormatExtension {
+	// The delimiter has the '^' prepended to the regex where the pattern is compiled.
```

Comment:
It took me a while to understand what this comment was saying, so I wonder if it would be clearer if it was worded something like this:

> // '^' is prepended to the regex in LICENSE_HEADER_DELIMITER later in FormatExtension.licenseHeader(String, String)

Reply:
Sure, sounds good.